### PR TITLE
NMEDocReorg

### PR DESCRIPTION
**Pull request by PirateJC**

### CODE REVIEW

**Comment by PirateJC** [comment]
@RaananW and @PatrickRyanMS can you take a look at this PR. This is the re-organization of the NME doc into several different pages.

Also @RaananW - The build failed for some reason. Any thoughts?

**Review by PatrickRyanMS — APPROVED** [review]

File: configuration/structure.json [hunk 1]
```diff
@@ -269,6 +269,11 @@
                             "friendlyName": "Node Materials",
                             "children": {
                                 "nodeMaterial": { "friendlyName": "Node Materials", "children": {}, "content": "How_To/material/Node_Material" },
+                                "NMEPBR": { "friendlyName": "Node Material and PBR", "children": {}, "content": "How_To/material/NME_PBR" },
+                                "NMEPostProcesses": { "friendlyName": "Node Material Post Processes", "children": {}, "content": "How_To/material/NME_Post_processes" },
```
Comment (RaananW) [inline configuration/structure.json]:
```suggestion
                                "NMEPostProcesses": { "friendlyName": "Node Material Post Processes", "children": {}, "content": "How_To/material/NME_Post_Processes" },
```

**Review by RaananW — APPROVED** [review]

**Comment by RaananW** [comment]
> Also @RaananW - The build failed for some reason. Any thoughts?

The error is `no such file or directory, stat 'content/How_To/material/NME_Post_processes.md'`

Should be resolved now.

File: configuration/structure.json [hunk 2]
```diff
@@ -269,6 +269,11 @@
                             "friendlyName": "Node Materials",
                             "children": {
                                 "nodeMaterial": { "friendlyName": "Node Materials", "children": {}, "content": "How_To/material/Node_Material" },
+                                "NMEPBR": { "friendlyName": "Node Material and PBR", "children": {}, "content": "How_To/material/NME_PBR" },
+                                "NMEPostProcesses": { "friendlyName": "Node Material Post Processes", "children": {}, "content": "How_To/material/NME_Post_Processes" },
+                                "NMEProceduralTexture": { "friendlyName": "Node Material Procedural Textures", "children": {}, "content": "How_To/material/NME_Procedural_Textures" },
+                                "NMEParticleShaders": { "friendlyName": "Node Material Particle Shaders", "children": {}, "content": "How_To/material/NME_Particle_Shaders" },
+                                "availableNodeBlocks": { "friendlyName": "Available Nodes (blocks)", "children": {}, "content": "How_To/material/Available_nodes" },
```
Comment (RaananW) [inline configuration/structure.json]:
```suggestion
                                "availableNodeBlocks": { "friendlyName": "Available Nodes (blocks)", "children": {}, "content": "How_To/material/Available_Nodes" },
```

**Pull request merged**